src/MetricsParser.py

```diff
@@ -7,6 +7,7 @@
 PICARD_COLLECT_WGS_METRICS = "picard_CollectWgsMetrics"
 FASTQC = "fastqc"
 NANOPLOT = "nanoplot"
+VERIFYBAMID = "verifyBamID"
```

Comment:
```suggestion
VERIFYBAMID = "verifybamid2"
```

Reply:
I would probably keep it lowerkey like the other software we are mentioning and add the 2 since it's technically VerifyBAMID2

Reply:
Works for me